src/jit/flowgraph.cpp

```diff
@@ -5401,9 +5401,17 @@ void Compiler::fgMakeBasicBlocks(const BYTE* codeAddr, IL_OFFSET codeSize, BYTE*
                 jmpKind = BBJ_EHFINALLYRET;
                 break;
 
+            case CEE_TAILCALL:
+                if (compIsForInlining())
+                {
+                    // TODO-CQ: We can inline some callees with explicit tail calls if we can guarantee that the calls
+                    // can be dispatched as tail calls from the caller.
+                    compInlineResult->NoteFatal(InlineObservation::CALLEE_EXPLICIT_TAIL_PREFIX);
+                    return;
+                }
+                // Intentionally fall through
             case CEE_READONLY:
```

Comment:
Note that we typically use:

 __fallthrough;

with surrounding blank lines for emphasis.